scripts/remotedev.yml

```diff
@@ -0,0 +1,31 @@
+Description: >
```

Comment:
This is the CloudFormation entrypoint that the rdev script uses to create/update dev stacks